db/migrate/20190721113740_create_restaurants.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRestaurants < ActiveRecord::Migration[5.1]
+  def change
+    create_table :restaurants do |t|
```

Comment:
Just in general, we follow specific guidelines for migrations
You don't need to change it, just so you know
Instead of using `change`, we define both `up` and `down`, even though the `down` is auto-generated by rails if you don't define it for simple cases like this. This is just so we have more clarity what exactly happens.
We also add checks of `unless table_exists?` when creating tables. This sometimes helps in environments where there's a mismatch between the DB and the migrations checked as run (fails CircleCI for example sometimes).